packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -208,7 +208,7 @@ class EditableText extends StatefulWidget {
     this.onSelectionChanged,
     List<TextInputFormatter> inputFormatters,
     this.rendererIgnoresPointer = false,
-    this.cursorWidth = 1.0,
+    this.cursorWidth = 2.0,
```

Comment:
This is a backwards incompatible change. Is it really needed?

Reply:
[per discussion with @sandrasandeep] If the native textfields all use 2.0 wide cursors, and Material doesn't specify a text cursor width, then it makes sense to bite the bullet and make the default 2.0.

So although this is an incompatible change, it's probably best to go though with it now, while Flutter is still in beta.